boards/nxp/ucans32k146/src/can_boot.c

```diff
@@ -208,6 +232,7 @@ static const  led_t i2l[] = {
 	led(9, fw_update_timeout,              31,    0,    0,     2),
 	led(a, fw_update_invalid_crc,          31,    0,    0,     4),
 	led(b, jump_to_app,                     0,   63,    0,    10),
+	led(c, hardware_failure,               31,    0,    0,    10),
```

Comment:
Why not 63? So it is really bright, because it is really broken! :)

Reply:
So 31 is the max value since red is encoded to use 5 bits see
https://github.com/PX4/PX4-Autopilot/blob/e972fd82e5c106be7bf6969567e25ac94ec82611/boards/nxp/ucans32k146/src/can_boot.c#L214-L219

Reply:
Ahh yes... I had done the 565 for a super constrained flash to fit the code. Is that needed here?

Reply:
We've got 1MB of flash on the S32K146, however adding extra bits to the rgb wouldn't change much in color clarity since we're using a quite small rgb led on UCANS32K146 anyhow, which makes it hard to spot any difference.